src/migrations/user-store/file-name-migration.ts

```diff
@@ -23,18 +23,18 @@ import fse from "fs-extra";
 import { app, remote } from "electron";
 import path from "path";
 
-export async function fileNameMigration() {
+export function fileNameMigration() {
```

Comment:
Making this synchronous allows this to be done within the `UserStore`'s `constructor`.